### PR TITLE
Create main_sv.ts

### PR DESCRIPTION
**Pull request by eson57**
Add Swedish translation please.

### CODE REVIEW

**Comment by KOV70** [comment]
Hello! Please change in installer_ru.ts шведский to Шведский

File: src/translations/main_sv.ts [hunk 1]
```diff
@@ -0,0 +1,812 @@
...
+        <source>Minimum notification font size:</source>
+        <translation>Minsta teckenstorlek för avisering:</translation>
+    </message>
+    <message>
+        <source> points</source>
+        <translation> punkter</translation>
+    </message>
+    <message>
+        <source>When blinking, use the fade-in/fade-out transition (increases CPU use!)</source>
+        <translation>Använd in-/uttoning vid blinkning (ökad cpu-belastning)</translation>
```
Comment (Abestanis) [inline src/translations/main_sv.ts:306]:
```suggestion
        <translation>Använd in-/uttoning vid blinkning (ökad CPU-belastning)</translation>
```
Is it correct to write the abbreviation in lowercase letters? I'm not sure, if this is correct in Swedish then please ignore this suggestion.

File: src/translations/main_sv.ts [hunk 2]
```diff
@@ -0,0 +1,812 @@
...
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;Noto Sans&apos;; font-size:8pt;&quot;&gt;Copyright (C) 2018 by George Yunaev, &lt;/span&gt;&lt;a href=&quot;mailto:gyunaev@ulduzsoft.com&quot;&gt;&lt;span style=&quot; font-family:&apos;Noto Sans&apos;; font-size:8pt; text-decoration: underline; color:#2980b9;&quot;&gt;gyunaev@ulduzsoft.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:&apos;Noto Sans&apos;; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;Noto Sans&apos;; font-size:8pt;&quot;&gt;Birdtray ir FRI MJUKVARA, som är licensierad under General Public License v3. För att förtydliga ytterligare kan du använda den för alla ändamål, inklusive kommersiella, utan att betala något. &lt;/span&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_sv.ts:458]:
```suggestion
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:&apos;Noto Sans&apos;; font-size:8pt;&quot;&gt;Birdtray ir FRI MJUKVARA, som är licensierad under General Public License v3. För att förtydliga ytterligare kan du använda den för alla ändamål, inklusive kommersiella, utan att betala något.&lt;/span&gt;&lt;/p&gt;
```

File: src/translations/main_sv.ts [hunk 3]
```diff
@@ -0,0 +1,812 @@
...
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Om det är aktiverat, lägger det här alternativet till &amp;quot;Ignorera för närvarande olästa e-postmeddelanden&amp;quot; i snabbmenyn. Den här åtgärden låter dig ignorera de e-postmeddelanden som för närvarande är olästa. Birdtray kommer då att låtsas att det inte finns några olästa e-postmeddelanden kvar och bara visa nya meddelanden som inkommer efter det ignorerade antalet.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Om det till exempel finns 10 olästa e-postmeddelanden och du klickar på &amp;quot;Ignorera&amp;quot;, visar Birdtray ingen oläst e-post så länge antalet olästa e-postmeddelanden ligger kvar på 10. När sedan ny e-post tas emot och du har totalt 11 olästa e-postmeddelanden, kommer Birdtray att visa  1 på räkneverket.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Om antalet olästa e-postmeddelanden går under det &amp;quot;ignorerade&amp;quot; antalet, återställs antalet ignorerade till noll&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
[the remark this answers is not in the document]
Comment (Abestanis) [reply src/translations/main_sv.ts:432]: There were two white-spaces `visa` and `1`.

**Pull request merged**